tests/test_cache.py

```diff
@@ -55,129 +61,129 @@ def tearDown(self):
-        assert Addon.objects.filter(id=1)[0].from_cache is True
+        self.assertIs(Addon.objects.filter(id=1)[0].from_cache, False)
+        self.assertIs(Addon.objects.filter(id=1)[0].from_cache, True)
 
     def test_slice_cache(self):
-        assert Addon.objects.filter(id=1)[:1][0].from_cache is False
-        assert Addon.objects.filter(id=1)[:1][0].from_cache is True
+        self.assertIs(Addon.objects.filter(id=1)[:1][0].from_cache, False)
+        self.assertIs(Addon.objects.filter(id=1)[:1][0].from_cache, True)
 
     def test_invalidation(self):
-        assert Addon.objects.get(id=1).from_cache is False
+        self.assertIs(Addon.objects.get(id=1).from_cache, False)
         a = [x for x in Addon.objects.all() if x.id == 1][0]
-        assert a.from_cache is False
+        self.assertIs(a.from_cache, False)
 
-        assert Addon.objects.get(id=1).from_cache is True
+        self.assertIs(Addon.objects.get(id=1).from_cache, True)
         a = [x for x in Addon.objects.all() if x.id == 1][0]
-        assert a.from_cache is True
+        self.assertIs(a.from_cache, True)
 
         a.save()
-        assert Addon.objects.get(id=1).from_cache is False
+        self.assertIs(Addon.objects.get(id=1).from_cache, False)
         a = [x for x in Addon.objects.all() if x.id == 1][0]
-        assert a.from_cache is False
+        self.assertIs(a.from_cache, False)
 
-        assert Addon.objects.get(id=1).from_cache is True
+        self.assertIs(Addon.objects.get(id=1).from_cache, True)
         a = [x for x in Addon.objects.all() if x.id == 1][0]
-        assert a.from_cache is True
+        self.assertIs(a.from_cache, True)
 
     def test_invalidation_cross_locale(self):
-        assert Addon.objects.get(id=1).from_cache is False
+        self.assertIs(Addon.objects.get(id=1).from_cache, False)
         a = [x for x in Addon.objects.all() if x.id == 1][0]
-        assert a.from_cache is False
+        self.assertIs(a.from_cache, False)
 
-        assert Addon.objects.get(id=1).from_cache is True
+        self.assertIs(Addon.objects.get(id=1).from_cache, True)
         a = [x for x in Addon.objects.all() if x.id == 1][0]
-        assert a.from_cache is True
+        self.assertIs(a.from_cache, True)
 
         # Do query & invalidation in a different locale.
         old_locale = translation.get_language()
         translation.activate('fr')
-        assert Addon.objects.get(id=1).from_cache is True
+        self.assertIs(Addon.objects.get(id=1).from_cache, True)
         a = [x for x in Addon.objects.all() if x.id == 1][0]
-        assert a.from_cache is True
+        self.assertIs(a.from_cache, True)
 
         a.save()
 
         translation.activate(old_locale)
-        assert Addon.objects.get(id=1).from_cache is False
+        self.assertIs(Addon.objects.get(id=1).from_cache, False)
         a = [x for x in Addon.objects.all() if x.id == 1][0]
-        assert a.from_cache is False
+        self.assertIs(a.from_cache, False)
```

Comment:
If I get the time and motivation, I may take this PR and tweak it to use those assert functions instead then.

Thanks!